### DISCUSSION TITLE
Architecture documentation

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-native#1647] [Q&A] [answered]
Is https://maplibre.org/maplibre-native/docs/book/design/media/proposed-architecture-of-maplibre-gl.png still up-to-date?

**Comment by louwers** [comment, marked as answer]
It will be in 10 years. 😄

**Reply by wipfli** [reply]
Uff you are right... What about the rest of the doc, is there anything that needs updating after the modularization?
